services/user.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+User = get_user_model()
+
+
+def create_user(
+        username: str, password: str, email: str | None = None,
+        first_name: str | None = None, last_name: str | None = None
+) -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email if email is not None else "",
+        first_name=first_name if first_name is not None else "",
+        last_name=last_name if last_name is not None else ""
```

Comment:
```suggestion
        email=email or "",
        first_name=first_name or "",
        last_name=last_name or ""
```
better use or in this case